### PR TITLE
Fix `werkzeug` 2.1.0 import and `skip` calculation

### LINKED ISSUE #83: ImportError: cannot import name 'get_current_traceback' from 'werkzeug.debug.tbtools'
Running into a similar issue as in plotly/dash#1992:

```
Traceback:
/usr/local/lib/python3.7/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
vizx_dashboards/__init__.py:26: in <module>
    from .create_dashboard import main as create_dashboard  # noqa: F401
vizx_dashboards/create_dashboard.py:22: in <module>
    from jupyter_dash import JupyterDash
../../../brix_venv/vizx_dashboards/lib/python3.7/site-packages/jupyter_dash/__init__.py:2: in <module>
    from .jupyter_app import JupyterDash
../../../brix_venv/vizx_dashboards/lib/python3.7/site-packages/jupyter_dash/jupyter_app.py:21: in <module>
    from werkzeug.debug.tbtools import get_current_traceback
E   ImportError: cannot import name 'get_current_traceback' from 'werkzeug.debug.tbtools' (/home/********/brix_venv/vizx_dashboards/lib/python3.7/site-packages/werkzeug/debug/tbtools.py)
```

### PR DESCRIPTION
**Pull request by deepyaman**
Closes #83

### CODE REVIEW

File: jupyter_dash/jupyter_app.py [hunk 1]
```diff
@@ -356,18 +378,12 @@ def _config_callback_exception_handling(
     ):
 
         @self.server.errorhandler(Exception)
-        def _wrap_errors(_):
+        def _wrap_errors(error):
             """Install traceback handling for callbacks"""
             self._traceback = sys.exc_info()[2]
 
             # Compute number of stack frames to skip to get down to callback
```
Comment (T4rk1n) [inline jupyter_dash/jupyter_app.py:385]: You have the traceback here or in `error.__traceback__`, can extract the text with the `traceback` module.

File: jupyter_dash/jupyter_app.py [hunk 2]
```diff
@@ -17,10 +17,32 @@
 from ansi2html import Ansi2HTMLConverter
 import uuid
 
+from .comms import _dash_comm, _jupyter_config, _request_jupyter_config
 
-from werkzeug.debug.tbtools import get_current_traceback
 
-from .comms import _dash_comm, _jupyter_config, _request_jupyter_config
+def _get_skip(error: Exception, divider=2):
```
Comment (T4rk1n) [inline jupyter_dash/jupyter_app.py:23]: Divider can be removed if refactor to raw traceback as it will be 1.

**Review by T4rk1n — COMMENTED** [review]
In dash we used the werkzeug debugger html in the dev tools ui. As it not used in here, we can remove that import and just use `traceback` module to get stacktrace.

File: jupyter_dash/jupyter_app.py (hunk 2 above)
Comment (deepyaman) [reply jupyter_dash/jupyter_app.py]: If I understand correctly, the `divider` is always 1 for raw traceback, as it is [here](https://github.com/plotly/dash/blob/5e89d95225d17e4ae4b0fdd94fd1c05d7bfe8761/dash/dash.py#L139). I've updated the code accordingly.

File: jupyter_dash/jupyter_app.py (hunk 1 above)
Comment (deepyaman) [reply jupyter_dash/jupyter_app.py:371]: Got it!

File: jupyter_dash/jupyter_app.py [hunk 3]
```diff
@@ -17,10 +17,32 @@
...
+        tbtools = None
+
+    # werkzeug<2.1.0
+    if hasattr(tbtools, "get_current_traceback"):
+        tb = tbtools.get_current_traceback()
+        text = tb.plaintext.splitlines()
+
+    if hasattr(tbtools, "DebugTraceback"):
+        tb = tbtools.DebugTraceback(error)  # pylint: disable=no-member
+        text = tb.render_traceback_text().splitlines()
```
[the remark this answers is not in the document]
Comment (deepyaman) [reply jupyter_dash/jupyter_app.py]: Done1

**Comment by deepyaman** [comment]
> In dash we used the werkzeug debugger html in the dev tools ui. As it not used in here, we can remove that import and just use `traceback` module to get stacktrace.

Thank you for the explanation! Think I've made the changes accordingly.

**Review by T4rk1n — APPROVED** [review]
:woman_dancing:  Looks good, thank you.

**Comment by alexcjohnson** [comment]
Thank you @deepyaman 🙇

**Pull request merged**